app/javascript/react/screens/App/Plan/components/PlanRequestDetailList.js

```diff
@@ -24,6 +24,75 @@ import sortFilter from './sortFilter';
 import paginate from './paginate';
 
 class PlanRequestDetailList extends React.Component {
+  static getDerivedStateFromProps(nextProps) {
+    const filterTypes = nextProps.planFinished
+      ? [
```

Comment:
do you mind pulling the filter types arrays out into constants? (i.e. maybe into PlanRequestDetailListConstants.js or something?)